prow/jobs/kyma-project/eventing-publisher-proxy/eventing-publisher-proxy-generic.yaml

```diff
@@ -63,7 +63,7 @@ postsubmits: # runs on main
         prow.k8s.io/pubsub.topic: "prowjobs"
         preset-sa-kyma-push-images: "true"
         preset-signify-prod-secret: "true"
-      run_if_changed: '^Dockerfile|\.go$|^go.(sum|mod)'
+      always_run: true
```

Comment:
Can you also add instructions to only run this job on [main](https://github.com/kyma-project/test-infra/blob/72faa7b0513b90be86d1538d7f1d146ab73ce038/prow/jobs/kyma-project/eventing-manager/eventing-manager-generic.yaml#L70). Background, we tag the image that gets generated here as `latest` and we only want images from main to have this tag.